web/views.py

```diff
@@ -461,3 +464,45 @@ def concept_reference(concept_id, name, lang):
         "code": format_code_for_display(concept_id, lang),
         "comment": format_comment_for_display(concept_id, lang)
     }
+
+
+# API functions
+
+def api(request, lang, version, structure):
+    """
+    Returns the template for a language, version, and structure
+
+    :param lang: language to get the template for
+    :param version: version of the language
+    :param structure: structure of the language
+    :return: HttpResponse object with rendered object of the page
+    :throws: HttpResponseNotFound if the language, version, or structure is not found
+    """
+    store_url_info(request)
+
+    file_path = os.path.join(
+        os.path.dirname(__file__),
+        "thesauruses",
+        lang,
+        version,
+        structure + ".json"
+    )
+
+    try:
+        with open(file_path, "r") as file:
+            response = file.read()
+    except FileNotFoundError:
+        return HttpResponseNotFound()
```

Comment:
I would be much more comfortable with this if file handling stayed in `models.py`